docker_cli/cli/build_commands.py

```diff
@@ -55,12 +56,38 @@ def init_build_parsers(subparsers: argparse._SubParsersAction) -> None:
     )
     add_tag_argument(parser=archive_parser, default="repo")
 
+    copy_dir_parser = subparsers.add_parser(
+        "copydir",
+        parents=[setup_parse],
+        help="Insert the contents of a directory at the given path.",
+        formatter_class=help_formatter,
+    )
+    add_tag_argument(parser=copy_dir_parser, default="dir-copy")
+    copy_dir_parser.add_argument(
+        "--directory",
+        "-d",
+        type=Path,
+        required=True,
+        help="The directory to be copied to the image.",
+    )
+    copy_dir_parser.add_argument(
+        "--target-path",
+        "-p",
+        type=Path,
+        default=None,
+        help="The path on the image to copy the source directory to. If not given, "
+        "the lowest level directory of the path given by the directory argument "
```

Comment:
Reading through the code, I had a hard time interpreting what you meant by "lowest level directory". I think "[base name](https://docs.python.org/3/library/os.path.html#os.path.basename)" might be the more commonly used term for this.

```suggestion
        "the base name of the path given by the directory argument "
```